src/components/Card/Card.tsx

```
@@ -1,65 +1,111 @@
 import Image from "next/image";
```

ESLint

[error] 'Image' is defined but never used.

(@typescript-eslint/no-unused-vars)